src/win/process.c

```diff
@@ -1081,6 +1081,10 @@ int uv_spawn(uv_loop_t* loop,
     process_flags |= DETACHED_PROCESS | CREATE_NEW_PROCESS_GROUP;
   }
 
+  /* Hide console windows */
+  if (options->flags & UV_PROCESS_WINDOWS_HIDE)
+    process_flags |= CREATE_NO_WINDOW;
```

Comment:
Suggestion: move the `process_flags` assignment on line 1068 up a few lines and then merge this with the `if` block on line 1061.

Reply:
I'll make that change. I originally thought about doing that anyway.